src/mappings.go

```diff
@@ -176,6 +139,19 @@ func convertMappingsJsonToMappings(json MappingsJson) (Mappings, error) {
 	return m, nil
 }
 
+func mergeMappingsFromDefault(dist *Mappings, platform string) error {
```

Comment:
> Shall I fix this as well?

I couldn't get a point. What do you have a problem with the line? (Or line was changed due to your later changes?)